compiler/parser/chapel.ypp

```diff
@@ -542,37 +542,41 @@ stmt:
   deprecated_decl_stmt
 | include_module_stmt
 | block_stmt
+| executable_stmt
 | use_stmt
 | import_stmt
 | require_stmt
-| assignment_stmt
 | extern_block_stmt
-| if_stmt
 | implements_stmt
 | interface_stmt
-| loop_stmt
-| select_stmt
 | defer_stmt
 | try_stmt
 | throw_stmt
 | return_stmt
+| TATOMIC stmt                         { $$ = buildAtomicStmt($2); }
+| TBREAK opt_label_ident TSEMI         { $$ = buildGotoStmt(GOTO_BREAK, $2); }
+| TCONTINUE opt_label_ident TSEMI      { $$ = buildGotoStmt(GOTO_CONTINUE, $2); }
+| TLABEL ident_def stmt                { $$ = buildLabelStmt($2, $3); }
+| TYIELD expr TSEMI                    { $$ = buildPrimitiveStmt(PRIM_YIELD, $2); }
+| error TSEMI                          { $$ = buildErrorStandin(); }
+;
+
+executable_stmt:
+  assignment_stmt
+| if_stmt
+| loop_stmt
+| select_stmt
 | manage_stmt
```

Comment:
👍 I was surprised when I tried to write `try! manage ...` and it didn't work. This will be really elegant.